### PR TITLE
chore: add get fee asset id to chain adapters

### PR DESCRIPTION
**Pull request by elmutt**
Add fee asset id to chain adapters to make it easier to get the fee asset for a given asset

### CODE REVIEW

File: packages/chain-adapters/src/api.ts [hunk 1]
```diff
@@ -8,6 +8,12 @@ export type ChainAdapter<T extends ChainTypes> = {
   getType(): T
 
   getChainId(): ChainId
+
+  /**
+   * Base fee asset used to pay for txs on a given chain
+   */
+  getFeeAssetId(): string
```
Comment (0xdef1cafe) [inline packages/chain-adapters/src/api.ts:15]:
```suggestion
  getFeeAssetId(): AssetId
```

File: packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts [hunk 2]
```diff
@@ -79,6 +79,10 @@ export class ChainAdapter
     return ChainTypes.Bitcoin
   }
 
+  getFeeAssetId(): string {
```
Comment (0xdef1cafe) [inline packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts:82]:
```suggestion
  getFeeAssetId(): AssetId {
```

File: packages/chain-adapters/src/cosmossdk/cosmos/CosmosChainAdapter.ts [hunk 3]
```diff
@@ -54,6 +54,10 @@ export class ChainAdapter extends CosmosSdkBaseAdapter<ChainTypes.Cosmos> {
     return ChainTypes.Cosmos
   }
 
+  getFeeAssetId(): string {
```
Comment (0xdef1cafe) [inline packages/chain-adapters/src/cosmossdk/cosmos/CosmosChainAdapter.ts:57]:
```suggestion
  getFeeAssetId(): AssetId {
```

File: packages/chain-adapters/src/ethereum/EthereumChainAdapter.ts [hunk 4]
```diff
@@ -72,6 +72,10 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Ethereum> {
     return this.chainId
   }
 
+  getFeeAssetId(): string {
```
Comment (0xdef1cafe) [inline packages/chain-adapters/src/ethereum/EthereumChainAdapter.ts:75]:
```suggestion
  getFeeAssetId(): AssetId {
```

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]
please fix commit history and PR description

**Review by DaoDev44 — DISMISSED** [review]

**Review by 0xdef1cafe — APPROVED** [review]

**Review by DaoDev44 — APPROVED** [review]

**Pull request merged**